source/scale/high-availability-cluster-based-deployment.rst

```diff
@@ -419,6 +419,28 @@ Copy all the above settings to the read replica, and modify or add only the belo
   hot_standby = on
   hot_standby_feedback = on
 
+**A note on vacuuming**
+
+Performance of a Postgres database is particularly sensitive to `vacuuming and analyzing <https://www.postgresql.org/docs/current/sql-vacuum.html>`__. A good way to check how frequently tables are vacuumed is with this query:
+
+.. code-block:: sql
+
+  SELECT relname, n_tup_ins as inserts,n_tup_upd as updates,n_tup_del as deletes, n_live_tup as live_tuples, n_dead_tup as dead_tuples, n_mod_since_analyze, last_autovacuum, last_autoanalyze, autovacuum_count, autoanalyze_count FROM pg_stat_user_tables order by dead_tuples desc LIMIT 10;
+
+The output of this query will indicate which tables have accumulated the most dead tuples, and you can also look at the last_autovacuum and last_autoanalyze columns to see when did the last autovacuum or autoanalyze run.
+
+Depending on those value, you can choose to tune table-specific values for autovacuum or autoanalyze threshold. For example, if you see more than 50,000 dead tuples on a table, and it hasn't been vacuumed or analyzed in the last 6 months, there's a good chance that it would probably benefit from more aggressive vacuuming. In that case, you can run this to tune your tables:
+
+.. code-block:: sql
+
+  ALTER TABLE <table> SET (
+    autovacuum_vacuum_scale_factor = 0.1, -- default is 0.2
+    autovacuum_analyze_scale_factor = 0.05, -- default is 0.1
+    autovacuum_vacuum_cost_limit = 1000 -- default is 200
+  );
+
+Feel free to choose different values as necessary. Refer to https://www.postgresql.org/docs/current/routine-vacuuming.html#AUTOVACUUM for more information on how does Postgres calculate when to run vacuuming. Re-run the SQL query from time to time and adjust values accordingly.
```

Comment:
Once we are Postgres only, I'd love it if we could start thinking of ways of making auto vacuuming truly "auto" :)